glustercli/cmd/volume-reset.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	volumeResetCmdHelpShort = "Reset volume options"
+	volumeResetCmdHelpLong  = "Reset options on a specified gluster volume. Needs a volume name and at least one option"
+)
+
+var (
+	flagForce, flagResetAll bool
+
+	volumeResetCmd = &cobra.Command{
```

Comment:
For slightly better readability, please consider moving `volumeResetCmd` variable out of the `var` block.